templates/base.html

```diff
@@ -44,28 +45,28 @@
                         <!-- Nav -->
                         <div class="show-for-small-only">
                             <ul class="nav">
-                                <li><button type="button" class="menu-icon" data-toggle="offCanvas"></button></li>
+                                <li class="small-menu"><button type="button" class="menu-icon" data-toggle="offCanvas"></button></li>
                             </ul>
                         </div>
                         <div class="hide-for-small-only">
                             <ul class="nav">
                                 <li><a class="active" href="{% url 'home' %}">HOME</a></li>
-                                <li><a class="active" href="{% url 'artists' %}">ARTISTS</a></li>
+                                <li><a href="{% url 'artists' %}">ARTISTS</a></li>
                                 {% if user.is_authenticated %}
-                                    <li style="float: right;"><a class="active" href="{% url 'logout' %}">LOGOUT</a></li>
-                                    <li style="float: right;"><a class="active" href="{% url 'profile' %}">PROFILE</a></li>
+                                    <li style="float: right;"><a href="{% url 'logout' %}">LOGOUT</a></li>
+                                    <li style="float: right;"><a href="{% url 'profile' %}">PROFILE</a></li>
                                 {% else %}
                                     <li style="float: right;"><a data-open="login-modal">LOGIN</a></li>
+                                    <li style="float: right;"><a href="{% url 'register' %}">SIGN UP</a></li>
                                 {% endif %}
-                            </ul>
-                        </div>
+                        </ul>
```

Comment:
Looks like you unindented `</ul>` and `</div>` instead of indenting the Content section. Sorry for the ambiguity in my comment from before, but the Content section should be indented in.

Basically, the close `</ul>` should line up with the open `<ul>`. Same with the `</div>` and `<div class="hide-for-small-only">`. But also, the Content section should line up with the `<div class="hide-for-small-only">` section.